exporter/awsemfexporter/metric_translator.go

```diff
@@ -405,8 +405,8 @@ func translateCWMetricToEMF(cWMetric *cWMetrics, config *Config) *cwlogs.Event {
 					"Timestamp": 1668387032641
 			  	}
 			*/
-			fieldMap["Version"] = 0
-			fieldMap["Timestamp"] = cWMetric.timestampMs
+			fieldMap["Version"] = "0"
```

Comment:
nit: If you're going to have a comment with an example above, keep it updated with your changes. Make version and timestamp strings.